assets/js/renderer.js

```diff
@@ -388,8 +388,32 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Instantiate the splitter.
+    const splitter = Split(['#leftCol', '#tabbar'], {
+        sizes: [18, 76],
+        minSize: 200,
+        gutterSize: 5,
+        onDragEnd: function() {
+            sessionStorage.setItem('splitSizes', JSON.stringify(splitter.getSizes()));
+            barChart.draw();
```

Comment:
@jdaguil   I may not finalize this until tomorrow morning when I have a fresh set of eyes, but it looks like this redraw is triggered only when the user is viewing the Chart Summary View:

```
        onDragEnd: function() {
            sessionStorage.setItem('splitSizes', JSON.stringify(splitter.getSizes()));
            if ($('#bar-chart-container').is(':visible')) {
                barChart.draw();
            }
        }
```
All testing this evening is positive . . . .